### DISCUSSION TITLE
Node fails to start with "no logger is present"

### DISCUSSION THREAD

**Discussion by yujunchengg** [rabbitmq/rabbitmq-server#5383] [Other] [answered]
rabbitmq-server version 3.8.8
When I install the plug rabbitmq_delayed_message_exchange-3.8.0.ez success, and then restart the server, it restart failed and crazy screen swiping, the log is as follows
![image](https://user-images.githubusercontent.com/36986915/181707729-053b636b-9b92-40cc-99cc-2e7e08093949.png)

**Comment by michaelklishin** [comment, marked as answer]
RabbitMQ 3.8.8 is 27 patch releases behind the latest and a few days from [being completely out of support](rabbitmq.com/versions.html).

The message suggests that something tries to log when the logger isn't initialised, many times a second. 

My best guess is that you have downloaded the wrong (e.g. 3.9.x) or a really outdated version of the plugin. RabbitMQ has changed what logging library it uses starting with 3.9.0.

**Reply by yujunchengg** [reply]
I think the same version in the video is OK
